src/routes/BreedGallery.tsx

```diff
@@ -0,0 +1,18 @@
+import { useParams } from 'react-router-dom';
+import Header from '../components/Header/Header';
+import BreedPhotos from '../components/BreedPhotos/BreedPhotos';
+
+interface Params {
+    breed: string;
+}
+
+export default function BreedGallery(props: any) {
```

Comment:
No need to use props here, right?

Reply:
> Very nice work Kevin! Thank you so much for your time mate! can you please check my comments when you have some time? Thanks in advance and again thanks for your effort!

Thank you a lot for your kind words and valuable feedback! I replied to your comments. Two of them were duplicated and left those without reply.